lib/index.js

```diff
@@ -48,41 +77,133 @@ function compile_template(template_name, type) {
  * the email has been sent.
  */
 function sendemail(template_name, person, callback) {
-  // see: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SES.html#sendEmail-property
-  var params = {
-    Destination: { /* required */
-      ToAddresses: [
-        person.email
-      ]
-    },
-    Message: { /* required */
-      Body: { /* required */
-        Html: {
-          Data: compile_template(template_name, 'html')(person), /* required */
-          Charset: 'utf8'
+    // see: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SES.html#sendEmail-property
+    var params = {
+        Destination: { /* required */
+            ToAddresses: [
+                person.email
+            ]
         },
-        Text: {
-          Data: compile_template(template_name, 'txt')(person), /* required */
-          Charset: 'utf8'
+        Message: { /* required */
```

Comment:
changes to indentation is the _fastest_ way to get a PR rejected.
I know it's not your "_fault_" and it's _probably_ just your **editor** trying to "correct" things, but it's _definitely_ something to be _aware_ of.
We won't _accept_ a PR that _arbitrarily_ changes indentation from 2-space to 4-space without a valid reason. 👍